proc_width: 640
proc_height: 480
mm_per_px_x: 0.41
mm_per_px_y: 0.46
origin_px:
- 587.0
- 275.0
origin_mm:
- 148.69202330706582
- 365.8440789237809
- 623.4898940655077
- 19.937607757850362
swap_xy: true
invert_x: true
invert_y: true
theta_sign: 1
yaw_offset_deg: 15.0
plane_z_mm: 377.00
